### DISCUSSION TITLE
Check latest connection and latest upload for users

### DISCUSSION THREAD

**Discussion by jrhbcn** [immich-app/immich#2099] [Q&A]
Hi,

As the administrator of immich in my household, I was wondering if it is possible to know when a user has last accessed the server (through the android app) and when is the time of the latest uploaded photo. Some of the family members have phones which (after reading the help) are not very friendly to background processes. I wonder if I can check (manually or maybe automatically) if the app is still connecting and uploading photos after some time, so I can react and let them know to re-check the app. 

In the meantime, I thought abut checking the docker logs but I don't see any reference to new connections. Would it be possible?

Best regards and thanks for this piece of nice software,

**Comment by etnoy** [comment]
I don't think there is an endpoint for that, but if you are comfortable with SQL it could be possible to query the database directly

**Reply by jrhbcn** [reply]
I don't know much about SQL but I can try to see if I can manage to code something. Do you know if there is information of last access time of the mobile apps in the DB? Or it will only have the last uploaded file?

Thanks!
